examples/match_statements/src/main.sw

```diff
@@ -1,12 +1,11 @@
 script;
 
-fn foo() {
+fn foo() {}
     // do something
-}
-fn bar() {
-    // do something
-}
+fn bar() {}
 
+
+    // do something
```

Comment:
Comments are broken here

Reply:
I'll open an issue to address it, this happens when placing comments inside an empty block